packages/bpk-component-popover/src/BpkPopoverPortal-test.js

```diff
@@ -99,7 +100,7 @@ describe('BpkPopoverPortal', () => {
 
     it('should render correctly with portalStyle added to portal component', () => {
       const shallowRenderer = Shallow.createRenderer();
-      const customStyle = { color: 'red' };
+      const customStyle: ?Object = { color: 'red' };
```

Comment:
Why do we need to declare a type here? Per the [docs](https://flow.org/en/docs/types/objects/#toc-sealed-objects) flow should infer it

Reply:
I got an error in native `BpkIcon` and had to [explicitly declare an object](https://github.com/Skyscanner/backpack/blob/master/native/packages/react-native-bpk-component-icon/src/BpkIcon.js#L90), maybe this is the same error.

Reply:
Hmm okay but I think that's because you have an unsealed object there. This object is sealed. Btw you can improve the type in BpkIcon with `const icons: { [string]: string } = {};`

Reply:
Yup flow moaned until I added that type so ...

Reply:
Strange, any reason it's not `const customStyle: { color: string } = { color: 'red' };`

Reply:
Pure laziness ... i didn't think it was worth the bother given that it's a test file.

Reply:
@k0nserv you're right - my issue was due to an unsealed object. And thanks for the tip, I'll change that!